src/shaders/stdosl.h

```diff
@@ -530,7 +531,7 @@ int iscameraray () { return raytype("camera"); }
 int isdiffuseray () { return raytype("diffuse"); }
 int isglossyray () { return raytype("glossy"); }
 int isshadowray () { return raytype("shadow"); }
-int getmatrix (string fromspace, string tospace, output matrix M) BUILTIN;
+int getmatrix (string fromspace, string tospace, output matrix M) BUILTIN_NONSTANDARD_RW;
```

Comment:
The definition of `getmatrix` already contains a marked `output` parameter. Maybe not for this review, but would it be possible to directly use this information instead of having to manually flag the argument in the compiler?